lib/hypertrack.dart

```diff
@@ -53,49 +81,82 @@ class HyperTrack {
   /// In case of frequent changes, it is possible, that intermediate states
   /// could be lost, so if more real-time responsiveness is required it is
   /// recommended to use [addGeotag] for passing that data.
-  void setDeviceMetadata(Map<String, Object> data) =>
-      _pluginInterface.setDeviceMetadata(data);
+  Future<void> setMetadata(Map<String, Object> data) {
```

Comment:
No way to set metadata as strongly typed JSON?

Reply:
fixed